holo-bgp/src/packet/consts.rs

```diff
@@ -260,6 +262,7 @@ bitflags! {
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 #[derive(FromPrimitive, ToPrimitive)]
 #[derive(Deserialize, Serialize)]
+#[derive(Arbitrary)]
```

Comment:
unnecessary derive

Reply:
We use this in the following BGP fuzzes:

`as_path_segment_decode`, `aggregator_decode` and `as_path_decode`.